### PR TITLE
Feature/11 lesson 1 improve test using expected output

### PR DESCRIPTION
**Pull request by taitruong**
@KlugeCh @RagipGashi @RouvenGonzalez, please review.

### CODE REVIEW

File: src/lesson-001-iterations/001-binary-gap/l001t01.binary-gap.iteration.test.ts [hunk 1]
```diff
@@ -1,35 +1,100 @@
...
+    expect(output).toEqual(3);
+  });
+
+  it('binary with many 1s and decreasing gaps: 11000111001111011111', () => {
+    const output = solution(binaryToNumber('11000111001111011111'));
+    // expected gap
+    expect(output).toEqual(3);
+  });
+
+  it('huge gap 111111111111111111111111111111111111111111111111111', () => {
```
[the remark this answers is not in the document]
Comment (RouvenGonzalez) [reply src/lesson-001-iterations/001-binary-gap/l001t01.binary-gap.iteration.test.ts:95]: yes this should be huge binary, there is no gap

**Comment by KlugeCh** [comment]
Approved

File: src/lesson-001-iterations/001-binary-gap/l001t01.binary-gap.iteration.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (RouvenGonzalez) [reply src/lesson-001-iterations/001-binary-gap/l001t01.binary-gap.iteration.test.ts:95]: already fixt in [pr #16](https://github.com/software-developer-org/codility/pull/16).

**Comment by RouvenGonzalez** [comment]
Approved

**Pull request merged**